### ISSUE TITLE
Updating to 5.1.5 breaks Craft CLI

### ISSUE THREAD

**Issue by fvwanja** [solspace/craft3-freeform#1157]
**Describe the bug or issue you're experiencing**

**Steps to reproduce**

1. Being on Freeform version 5.1.2, run an update with `./craft update freeform`
2. Experience the following error

![Screenshot 2024-03-20 at 10 00 59](https://github.com/solspace/craft-freeform/assets/90781427/cb51b5dd-6e93-4632-9a6a-04e2ae811acc)

This also occurs, when I update the plugin via composer manually to 5.1.5. After `composer update` I can no longer run the Craft CLI:

Error output (I replaced the actual paths with `path-to-project`):

```
path-to-project git:(master) ✗ ./backend/app/craft

Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: yii\console\Response::setNoCacheHeaders()'

in /path-to-project/backend/app/vendor/yiisoft/yii2/base/Component.php:300

Stack trace:
#0 /path-to-project/backend/app/vendor/solspace/craft-freeform/packages/plugin/src/Bundles/Form/HiddenInputs/CsrfInput.php(22): yii\base\Component->__call('setNoCacheHeade...', Array)
#1 [internal function]: Solspace\Freeform\Bundles\Form\HiddenInputs\CsrfInput->__construct()
#2 /path-to-project/backend/app/vendor/yiisoft/yii2/di/Container.php(411): ReflectionClass->newInstanceArgs(Array)
#3 /path-to-project/backend/app/vendor/yiisoft/yii2/di/Container.php(186): yii\di\Container->build('Solspace\\Freefo...', Array, Array)
#4 /path-to-project/backend/app/vendor/solspace/craft-freeform/packages/plugin/src/Library/Bundles/BundleLoader.php(50): yii\di\Container->get('Solspace\\Freefo...')
#5 /path-to-project/backend/app/vendor/solspace/craft-freeform/packages/plugin/src/Freeform.php(723): Solspace\Freeform\Library\Bundles\BundleLoader::loadBundles('/path-to-project...')
#6 /path-to-project/backend/app/vendor/solspace/craft-freeform/packages/plugin/src/Freeform.php(233): Solspace\Freeform\Freeform->initBundles()
#7 /path-to-project/backend/app/vendor/yiisoft/yii2/base/BaseObject.php(109): Solspace\Freeform\Freeform->init()
#8 /path-to-project/backend/app/vendor/yiisoft/yii2/base/Module.php(161): yii\base\BaseObject->__construct(Array)
#9 /path-to-project/backend/app/vendor/craftcms/cms/src/base/Plugin.php(122): yii\base\Module->__construct('freeform', Object(craft\console\Application), Array)
#10 [internal function]: craft\base\Plugin->__construct('freeform', Object(craft\console\Application), Array)
#11 /path-to-project/backend/app/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#12 /path-to-project/backend/app/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('Solspace\\Freefo...', Array, Array)
#13 /path-to-project/backend/app/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('Solspace\\Freefo...', Array, Array)
#14 /path-to-project/backend/app/vendor/craftcms/cms/src/Craft.php(59): yii\BaseYii::createObject(Array, Array)
#15 /path-to-project/backend/app/vendor/craftcms/cms/src/services/Plugins.php(943): Craft::createObject(Array, Array)
#16 /path-to-project/backend/app/vendor/craftcms/cms/src/services/Plugins.php(228): craft\services\Plugins->createPlugin('freeform', Array)
#17 /path-to-project/backend/app/vendor/craftcms/cms/src/base/ApplicationTrait.php(1649): craft\services\Plugins->loadPlugins()
#18 /path-to-project/backend/app/vendor/craftcms/cms/src/console/Application.php(53): craft\console\Application->_postInit()
#19 /path-to-project/backend/app/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\console\Application->init()
#20 /path-to-project/backend/app/vendor/yiisoft/yii2/base/Application.php(204): yii\base\BaseObject->__construct(Array)
#21 /path-to-project/backend/app/vendor/yiisoft/yii2/console/Application.php(89): yii\base\Application->__construct(Array)
#22 [internal function]: yii\console\Application->__construct(Array)
#23 /path-to-project/backend/app/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#24 /path-to-project/backend/app/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('craft\\console\\A...', Array, Array)
#25 /path-to-project/backend/app/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('craft\\console\\A...', Array, Array)
#26 /path-to-project/backend/app/vendor/craftcms/cms/src/Craft.php(59): yii\BaseYii::createObject(Array, Array)
#27 /path-to-project/backend/app/vendor/craftcms/cms/bootstrap/bootstrap.php(250): Craft::createObject(Array)
#28 /path-to-project/backend/app/vendor/craftcms/cms/bootstrap/console.php(42): require('/path-to-project...')
#29 /path-to-project/backend/app/craft(12): require('/path-to-project...')
#30 {main}
```

**Expected behavior**

Just update

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.8.4
- Freeform Version: 5.1.2
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Additional context**

Updating from 5.1.2 to 5.1.4 works as expected.

**Comment by gustavs-gutmanis** [comment]
Sorry for this @fvwanja
A fix for this will be released later today. Meanwhile you could set the freeform dependency in `composer.json` to `dev-v5` and try again.

**Comment by Romanavr** [comment]
Same problem.

**Comment by simajkeee** [comment]
Same

**Comment by kjmartens** [comment]
Very sorry for the trouble everyone! We've just released a fix for this in Freeform 5.1.5.1. 🙂

**Comment by fvwanja** [comment]
Thank you! 👍
